openapi/components/schemas/Invoices/UpcomingInvoiceItem.yaml

```
@@ -0,0 +1,90 @@
+type: object
+readOnly: true
+properties:
+  id:
+    type: string
+    description: ID of the upcoming invoice item.
+    maxLength: 50
+    example: ii_0YVFDEQS2KCFTBN9HXWJFY55GV
+  isInterim:
+    description: Specifies if the line item gets into an interim invoice.
+    type: boolean
+  description:
+    description: Description of the upcoming invoice item.
+    type: string
+    maxLength: 1000
+  unitPrice:
+    description: Unit price of the upcoming invoice item.
+    type: number
+    format: double
+  quantity:
+    description: Quantity of the upcoming invoice item.
+    type: integer
+  price:
+    description: Total price of the upcoming invoice item.
+    type: number
+    format: double
+  productId:
+    type: string
+    nullable: true
+    description: ID of the product.
+    maxLength: 50
+    example: prod_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  planId:
+    type: string
+    nullable: true
+    description: ID of the plan.
+    maxLength: 50
+    example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
+  subscriptionId:
+    type: string
+    description: ID of the order.
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  discountAmount:
+    description: Discount amount applied to the upcoming invoice item.
+    type: number
+    format: double
+  periodStartTime:
+    description: Date and time when the billing period starts.
+    type: string
+    format: date-time
+  periodEndTime:
+    description: Date and time when the billing period ends.
+    type: string
+    format: date-time
+  periodNumber:
+    description: Consecutive number of the billing period which is covered by the upcoming invoice item.
```

nitpick: I don't understand want is meant by 'covered by'

I've made some suggestion here, please feel free to edit them.

```suggestion
    description: Consecutive number of billing periods in which the invoice item is included.
```

```suggestion
    description: Consecutive number of billing periods where the invoice item is added.
```